site/doc/config/reference.md

```diff
@@ -254,6 +254,20 @@ In this case the run step name will be the same of the command trimmed at the ma
 | environment | Map: EnvVarName(String) => EnvVarValue([Value](#value)) | Environment variables to set                                                       |
 | working_dir | String                                                  | Working dir where the entrypoint will be executed (also used for steps in a task)  |
 | user        | String                                                  | The user id or username to use when executing the entrypoint or the task run steps |
+| volumes     | List: [Volume](#Volume)                                 | A list of volumes                                                                  |
+
+###### Volume
+
+| Option | Type                                                            | Description   |
+| -------| --------------------------------------------------------------- | ------------- |
+| path   | String                                                          | Mountpoint    |
+| tmpfs  | Map: [TmpfsOption](#TmpfsOption) => TmpfsValue([Value](#value)) | tmpfs options |
```

Comment:
It's not a map but just a struct so it should be written as (with also name changes)
```
| tmpfs  | [TmpFS Volume](#tmpfs-volume)  | tmpfs volume |
```